### ISSUE TITLE
Compatibility with ISO-IDMP Standards

### ISSUE THREAD

**Issue by lindsmith** [phenopackets/phenopacket-schema#237]
Comment:
> Please ensure compatibility and interoperability with ISO-IDMP standards as well as with other already existing terminologies for coding some of the aspects mentioned in the "common schema" to define the ‘Phenopacket’ on page 6 of the paper.

Page 6 is the Table of Comments in the draft, but I think this commenter is referring to [this section](https://phenopacket-schema.readthedocs.io/en/latest/schema.html). As an ISO Project Lead, I should be able to access the [IDMP standards](https://www.ema.europa.eu/en/human-regulatory/overview/data-medicines-iso-idmp-standards-overview) free of charge, if used for the development of the ISO Phenopackets Draft. I will check/confirm.

**Comment by pnrobinson** [comment]
@lindsmith  any progress getting this documentation?

**Comment by lindsmith** [comment]
@pnrobinson  Yes! I have the information to request watermarked versions, though there are ~6 different standards to choose from and 4 of them have associated implementation guides. As a starting point, I'll request #2 and #3 because they seem to have the highest relevance to the v2 proposed fields, but let me know if others are of interest as well.

1. Health informatics — Identification of medicinal products — Data elements and structures for the unique identification and exchange of regulated information on substances
2. Health informatics — Identification of medicinal products — Data elements and structures for the unique identification and exchange of regulated information on pharmaceutical dose forms, units of presentation, routes of administration and packaging
3. Health informatics — Identification of medicinal products — Data elements and structures for the unique identification and exchange of units of measurement
4. Health informatics — Identification of medicinal products — Data elements and structures for the unique identification and exchange of regulated medicinal product information
5. Health informatics — Identification of medicinal products — Data elements and structures for unique identification and exchange of regulated pharmaceutical product information
6. Health informatics — Identification of medicinal products — Core principles for maintenance of identifiers and terms

**Comment by julesjacobsen** [comment]
@lindsmith Did you manage to get hold of 2 and 3?

**Comment by lindsmith** [comment]
@julesjacobsen Yes! Sent via email a while ago, but will resend now. I unfortunately can't post them here (too "public")

**Comment by julesjacobsen** [comment]
The `codeTermPair` `code` and `term` fields are directly comparable to the Phenopacket `OntologyClass` `id` and `label`, but other fields are not present in the `OntologyClass` although these are likely present in the ontology itself.
